db/versions/0072.yml

```diff
@@ -0,0 +1,176 @@
+version: 72
+description: add last_date_active to queue_workers
+migrationScript: |
+  begin
+    alter table queue_workers add column last_date_active timestamp with time zone not null;
```

Comment:
Nice catch, thanks! So, now the question is - what do we default this column as? Or just make it nullable?

Reply:
I think it's best to make it default null. 
Otherwise I'm not even sure which date to use there, we'd have to create a temporary table and fill it in with some dates from other tables? (`workers.created`)

Reply:
Yeah I'm on board with just making it null. @petemoore @djmitche you guys have thoughts?

Reply:
The downside of NULL is that, after the migration, you won't see that value again.  So all the conditionals to handle a null there will get very little exercise.  But, you're right that setting it to anything else makes the migration substantially more complex.

So, null makes sense, but maybe call it out as a possibility in the sql function definitions and in the API schema.